### PR TITLE
feat: create-nuget-package-action

### PR DESCRIPTION
**Pull request by benPearce1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
-# create-nuget-package-action
\ No newline at end of file
+# create-nuget-package-action
+
+<img alt= "" src="https://github.com/OctopusDeploy/create-nuget-package-action/raw/main/assets/github-actions-octopus.png" />
```
Comment (slewis74) [inline README.md:3]: I'm struggling to see this image, did we copy it across? We use the same image for all of the actions, so just copy the assets folder from one of the others.

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+This is a GitHub Action to create a NuGet package to push [Octopus Deploy](https://octopus.com/).
+
+## Examples
+
+Incorporate the following actions in your workflow to push a package to Octopus Deploy using an API key, a target instance (i.e. `server`), and a project:
```
Comment (slewis74) [inline README.md:9]: We didn't read this earlier, doesn't make sense for these actions (suspect this is wrong for the Zip one too, I'll fix it there.

File: README.md (hunk 1 above)
Comment (benPearce1) [reply README.md:3]: yeah, we missed it

**Review by slewis74 — APPROVED** [review]
Looks good. Couple of things with comments inline, and we just need to wait for the API client package build and release before one final update here back to the NPM version of the package

File: README.md [hunk 3]
```diff
@@ -1 +1,53 @@
...
+<img alt= "" src="https://github.com/OctopusDeploy/create-nuget-package-action/raw/main/assets/github-actions-octopus.png" />
+
+This is a GitHub Action to create a NuGet package to push [Octopus Deploy](https://octopus.com/).
+
+## Examples
+
+```yml
+steps:
+  - uses: actions/checkout@v3
+  - name: Install Octopus CLI 🐙
```
Comment (slewis74) [inline README.md:12]: we don't need the CLI anymore, we can remove this from the example

File: README.md [hunk 4]
```diff
@@ -1 +1,49 @@
-# create-nuget-package-action
\ No newline at end of file
+# create-nuget-package-action
+
+<img alt= "" src="https://github.com/OctopusDeploy/create-nuget-package-action/raw/main/assets/github-actions-octopus.png" />
+
+This is a GitHub Action to create a NuGet package to push [Octopus Deploy](https://octopus.com/).
```
Comment (slewis74) [inline README.md:5]:
```suggestion
This is a GitHub Action to create a NuGet package to push to [Octopus Deploy](https://octopus.com/).
```

**Pull request merged**